images/redis/conf/redis.conf

```
@@ -11,6 +11,4 @@ maxmemory-policy allkeys-lru
 protected-mode no
 bind 0.0.0.0
 
-# this disabled the persistent cache
-save ""
-appendonly no
\ No newline at end of file
+include /etc/redis/${flavor:ephemeral}.conf
```

shouldn't that be:
```suggestion
include /etc/redis/${FLAVOR:ephemeral}.conf
```
maybe nitpicking, but better readable

---

fixed in acf88dbcf152b4b67c33f6e9e46f203f95e78a2a